sblecdemo/src/main/java/com/nexenio/sblecdemo/DemoLayout.java

```diff
@@ -18,13 +20,12 @@
 
 import androidx.annotation.NonNull;
```

Comment:
Please be careful here, where are the androidx dependencies coming from? I don't find it in the gradle file

Reply:
There is the `android.useAndroidX=true` property in `gradle.properties`. AndroidX is the default now, this is how Android Studio created the project. See https://developer.android.com/jetpack/androidx/migrate